README.md

```diff
@@ -63,14 +63,12 @@ installed in order to build the frontend assets.
 You can directly use the `go` tool to download and install the `prometheus`
 and `promtool` binaries into your `GOPATH`:
 
-    $ GO111MODULE=on go get github.com/prometheus/prometheus/cmd/...
+    $ GO111MODULE=on go install github.com/prometheus/prometheus/cmd/...@latest
     $ prometheus --config.file=your_config.yml
 
-*However*, when using `go get` to build Prometheus, Prometheus will expect to be able to
```

Comment:
I think these old instructions were actually still mostly fine even for the new web UI. The files are still read from the directories as described, and either `make assets` or `make build` needs to have been called.